alexrallen/sonic-buildimage
# [Mellanox] Fix DPB supported breakout modes

#### Why I did it

Removing DPB breakout modes that require adjacent ports to be disabled as that is not supported by the current DPB infrastructure.

Correspondingly had to remove the `hwsku.json` file from any SKUs which utilized these removed modes such that the system will fall back to `ports_config.ini` and DPB will not be supported for those SKUs.

#### How I did it

Modified the `platform.json` files of Mellanox devices.

#### How to verify it

Execute `show int break [Ethernet]` on the affected platforms and ensure there are no modes present that would require an adjacent port to be disabled to function. 

#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog
[mellanox] Fix DPB supported breakout modes


#### A picture of a cute animal (not mandatory but encouraged)
![borzoi_body_image_2](https://user-images.githubusercontent.com/5898707/153689781-8d64724c-e9bc-476b-b84d-5c72d0d21648.jpg)
